templates_jinja2/team_list.html

```diff
@@ -22,6 +22,7 @@ <h1>Teams {{cur_page_label}} <small>{{ num_teams }} teams</small></h1>
       <div class="btn-group pull-right">
         <a class="btn btn-default" href="/avatars/2020"><span class="glyphicon glyphicon-user"></span> View  Avatars</>
         <a class="btn btn-default" href="/nearby?search_type=teams"><span class="glyphicon glyphicon-search"></span> Search Nearby</a>
+        <a class="btn btn-default" href="/teams/random"><span class="glyphicon glyphicon-random"></span> I'm Feeling Lucky</a>
```

Comment:
Personally, I'm not sure if this is the right place to put this, although admittingly I'm not sure where would be, other than just on the 404 page.  🤔

Reply:
It felt right to me, but I'm open to other ideas.